### PR TITLE
docs: expand Chinese comments for model utilities

### PR DESCRIPTION
**Pull request by ChaosWang666**
## Summary
- add detailed Chinese docstrings and parameter notes for KV cache implementations
- document base LLM APIs and model selection helper with Chinese descriptions

## Testing
- `pytest`

------
https://chatgpt.com/codex/tasks/task_e_68988ab7c5d08329b61a22cf9d021ea0

### CODE REVIEW

**Comment by vladislavkruglikov** [comment]
Hello @ChaosWang666

In the official ShadowKV implementation I found crucial bug regarding different logits between ShadowKVCache and ShadowKVCache_CPU as well as some other not optimal implementations. [Here](https://github.com/ByteDance-Seed/ShadowKV/issues/13#issuecomment-3274036980) I describe the bug. I found your fork as a promising work on top of ShadowKV and I am interested if you have fixed or encountered the same different logits as described in issue I linked
